### ISSUE TITLE
`onAppOpenAttribution` triggered without call `handleOpen(url, options: options)` from cold-start state

### ISSUE THREAD

**Issue by nil-biribiri** [AppsFlyerSDK/AppsFlyerFramework#168]
# Report

## SDK Version

6.2.5 (build 76)
 
## What did you do?
- Open App from a cold-start state with any deep-link

## What did you expect to happen?
- `onAppOpenAttribution` callback should get triggered only once

## What happened instead?
- `onAppOpenAttribution` callback got triggered twice.

## Please provide any other relevant information.
- I removed `handleOpen(url, options: options)` from AppDelegate 
`application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool` function
- Still got callback from `onAppOpenAttribution` but only in a cold-start state

**Comment by af-fess** [comment]
This issue has been inactive for a while, and will now be closed due to inactivity. If you still believe this is a valid issue, please feel free to reopen it or create a new issue with updated information.
